src/sql/workbench/contrib/notebook/browser/cellViews/codeCell.component.ts

```diff
@@ -39,13 +39,13 @@ export class CodeCellComponent extends CellView implements OnInit, OnChanges {
 	}
 
 	private _model: NotebookModel;
-	private _activeCellId: string;
+	protected _activeCellId: string;
```

Comment:
Convention is _ should only be for private vars, so please remove that for the ones you're updating

Reply:
Reverted this. I realized there was no need to make this protected as I could use get/set activeCellId. My mistake.